src/init.cpp

```diff
@@ -197,120 +207,352 @@ static void CreateNewConfigFile()
+                                "node is authorized for scraping and the scraper is activated (default: 0)",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::SCRAPER);
+    argsman.AddArg("-scraperkey=<address>", "Manually specify scraper public key in address form. This is not necessary "
+                                            "and will not work if the private key is not present in the scraper wallet file.",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::SCRAPER);
+
+    // Researcher
+    argsman.AddArg("-email=<email>", "Email address to use for CPID detection. Must match your BOINC account email",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::RESEARCHER);
+    argsman.AddArg("-boincdatadir=<path>", "Path to the BOINC data directory for CPID detection when the BOINC client uses"
+                                           " a non-default directory",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::RESEARCHER);
+    argsman.AddArg("-forcecpid=<cpid>", "Override automatic CPID detection with the specified CPID",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::RESEARCHER);
+    argsman.AddArg("-investor", "Disable CPID detection and do not participate in the research reward system",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::RESEARCHER);
+    argsman.AddArg("-pooloperator", "Skip pool CPID checks for staking nodes run by pool administrators",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::RESEARCHER);
+
+    // Wallet
+    argsman.AddArg("-upgradewallet", "Upgrade wallet to latest format",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
+    argsman.AddArg("-keypool=<n>", "Set key pool size to <n> (default: 100)",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
+    argsman.AddArg("-rescan", "Rescan the block chain for missing wallet transactions",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
+    argsman.AddArg("-salvagewallet", "Attempt to recover private keys from a corrupt wallet.dat",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
+    argsman.AddArg("-zapwallettxes", "Delete all wallet transactions and only recover those parts of the blockchain through"
+                                     " -rescan on startup",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
+    argsman.AddArg("-checkblocks=<n>", "How many blocks to check at startup (default: 2500, 0 = all)",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
+    argsman.AddArg("-checklevel=<n>", "How thorough the block verification is (0-6, default: 1)",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
+    argsman.AddArg("-loadblock=<file>", "Imports blocks from external blk000?.dat file",
+                   ArgsManager::ALLOW_ANY, OptionsCategory::WALLET);
```

Comment:
```suggestion
```

The `-loadblock` option is added twice. I think we should keep the one in `OptionsCategory::OPTIONS` since it's not really related to the wallet. On that note, `-checkblocks` and `-checklevel` probably make more sense in `OPTIONS` too.